client/src/components/CartHeader/CartHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import { Store } from "src/interfaces";
+import {
+  DEFAULT_CART_HEADER_TITLE,
+  DEFAULT_STORE_HEADER_SUBTITLE,
+} from "src/constants";
+import { Typography, Button } from "@material-ui/core";
+import Header from "src/components/Header";
+
+function CartHeader({
+  store,
+  scanBarcodeCallback,
+}: {
+  store: Store;
+  scanBarcodeCallback: () => Promise<void>;
+}) {
+  return (
+    <div className="CartHeader">
+      {store && (
```

Comment:
store doesn't seem to be optional variable, do you need to check it's existence?

Reply:
added this fix unintentionally in #156